ambari/pom.xml

```diff
@@ -43,6 +43,12 @@
         </dependency>
 
         <dependency>
+            <groupId>com.squareup.retrofit</groupId>
+            <artifactId>retrofit</artifactId>
+            <version>1.9.0</version>
+        </dependency>
+
```

Comment:
we should add this to jar otherwise we need to give instructions to also include it in lib/dropins